src/server/mod.rs

```diff
@@ -88,24 +88,22 @@ impl Server {
     /// initialize a new instance of the server from [Settings]
     pub async fn with_settings(settings: Settings) -> Result<dev::Server, HandlerError> {
         let filter = HandlerResult::<AdmFilter>::from(&settings)?;
+        let metrics = metrics_from_opts(&settings)?;
+        let tiles_cache = cache::TilesCache::new(384);
```

Comment:
384 Seems a bit specific. What's the source of this? Should it be a setting?

Reply:
I looked at the size of the cache yesterday and it was =~ 350, so just something bigger than the original 75. Then as of this morning it's more like =~ 550.

It's not terribly important (just preferably something larger than the default of 8 or whatever it is) so I don't think it warrants a setting.

Reply:
Fair.  Maybe add a comment to that effect like
```
// This value is arbitrary and based on the expected mean. Feel free to adjust.
```
Or make it a `const` so that it's just a bit less magical feeling. I can see this becoming a legendary value for stupid reasons.